docs/app/docs/guide/syntax-highlighting/page.mdx

```diff
@@ -170,18 +169,15 @@ export async function ANSI() {
   [42;1;39;0m PASS [0;32m Waiting for file changes...[0m
          [0;2mpress [0;1mh[0;2m to show help, press [0;1mq[0;2m to quit
 \`\`\``
-  const { result } = await compileMdx(`
-You can highlight ANSI escape codes:
-
-~~~md filename="Markdown"
+  const { result } = await compileMdx(`~~~md filename="Markdown"
```

Comment:
we had duplicate 
<img width="798" alt="image" src="https://github.com/user-attachments/assets/dd9e85df-7464-4076-b451-202383c35487">